helm/values.yaml

```
@@ -275,7 +275,10 @@ coder:
   # coder.command -- The command to use when running the Coder container. Used
   # for customizing the location of the `coder` binary in your image.
   command:
-    - /opt/coder
+    - /opt/coder server
```

Command should just be the program to run, like `/bin/sh` or `/opt/coder` without any args.

```suggestion
    - /opt/coder
```